src/dto/em-submission-access.params.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsOptional } from 'class-validator';
+
+import { Status } from '../enums/status.enum';
+import { Type } from 'class-transformer';
+
+export class EmSubmissionAccessParamsDTO {
+  @IsOptional()
+  @ApiProperty()
+  @Type(() => Number)
+  facilityId?: number;
+
+  @IsOptional()
+  @ApiProperty()
+  monitorPlanId?: string;
+
+  @IsOptional()
+  @ApiProperty()
+  @Type(() => Number)
+  year?: number;
+
+  @IsOptional()
+  @ApiProperty()
+  @Type(() => Number)
+  quarter?: number;
+
+  @IsOptional()
+  @ApiProperty({ enum: Status })
+  status?: string;
+}
```

Comment:
There are no required query-param. Just want to make sure all optional.

Reply:
Yup no required parameters for the API. After talking with Jason, he wanted to keep them all optional on the API. We will require a parameter to be selected on the UI

Reply:
Make sense.